content/docs/tech-resources/enabling-https-engagement-tracking-on-sparkpost.md

```diff
@@ -338,72 +338,72 @@ Once your CNAME is set up with your DNS provider, instead of providing an existi
 1. Follow [these steps](#switch-tracking-domain-to-secure-and-validate) to update and verify your tracking domain, as this requires the certificate to be present and valid.
 
 ---
-## Step by Step Guide with Fastly
+## Step-by-Step Guide with Fastly
 
-Sign up for Fastly or log in to an existing account.
+Sign up for [Fastly](https://www.fastly.com/) or log in to an existing account.
 
-1. Select the **Configure** tab on the Dashboard, then "Create Service". Give your service a name, and add your tracking domain under "Domains".
+1. Select the **Deliver** tab on the Dashboard, then click the **Create a delivery service** button. Give your service a name in the _Options_ menu and clicking **Edit service name**.
 
-    ![](media/enabling-https-engagement-tracking-on-sparkpost/fastly-create-service.png)
+     ![](media/enabling-https-engagement-tracking-on-sparkpost/fastly-2023-create-a-delivery-service.png)
 
+1. In the **Domains** section, insert your tracking domain into the provided field and click **Add**. A subdomain (like *click*.domain.com, rather than just domain.com) is recommended.
 
-1. Select "Origins" on the left. Add the correct endpoint address for your service, see [here](#sparkpost-tracking-endpoints).
+     ![](media/enabling-https-engagement-tracking-on-sparkpost/fastly-2023-add-domain.png)
 
-    ![](media/enabling-https-engagement-tracking-on-sparkpost/fastly-origin-hosts.png)
+1. Select **Origins** in the left-side menu. In the **Hosts** section, add the correct tracking endpoint for your service (also known as hostname), see possible values [here](#sparkpost-tracking-endpoints).
+     
+     ![](media/enabling-https-engagement-tracking-on-sparkpost/fastly-2023-add-host.png)
 
-    Fastly detects that SparkPost supports TLS, and shows the host entry like this. Optionally you can use the "pencil" edit icon to set a meaningful name.
+     Fastly detects that SparkPost supports TLS, and shows the host entry like below. Optionally you can use the "pencil" edit icon to set a meaningful name.
 
-    ![](media/enabling-https-engagement-tracking-on-sparkpost/fastly-origin.png)
+     ![](media/enabling-https-engagement-tracking-on-sparkpost/fastly-2023-host-added.png)
 
-    Fastly default settings pass the `user_agent` and `ip_address` through to SparkPost engagement tracking as expected.
+     Fastly forwards the `user_agent` and `host` HTTP headers by default to SparkPost engagement tracking as expected.
```

Comment:
Minor thing, I know it was already like this, but while we're at it, can we change `user_agent` to `User-Agent`?